components/econet/econet.cpp

```diff
@@ -183,6 +183,7 @@ void Econet::make_request() {
       str_ids.push_back("EVAPTEMP");
       str_ids.push_back("SUCTIONT");
       str_ids.push_back("DISCTEMP");
+      str_ids.push_back("ALRMALRT");
```

Comment:
Have you verified nothing breaks when you request this? I worry the whole request might fail if you specify an unsupported id.

Reply:
Yes, this is discussed in https://discord.com/channels/1148015790038188073/1148327349620838511/1150603800382554272

Currently, it returns a numerical value ; when there is no active alert it's `0.000000`.

Based on the on-device display, I'm pretty sure it will provide a count of active alerts (0-4) but it's possible it will just be 0 or 1.

Reply:
Requests do seem to fail with an incorrect ID. `ALRMALRT` works. `ALARM_01` which logs indicate should work results in request failure.

Once we have a log with an active alert we'll have to see how that plays out...

Reply:
The manual says: "Fault Indicator - Illuminated when the display detects OBJECT codes ALMCODE 1, 2, 3 or 4 is greater than "0". This indicates the Control Board has detected either an Alarm or an Alert."
Requesting ALMCODE1 to ALMCODE4 works. There is no data returned so we don't know their type yet but at least the request succeeds. Requesting ALMCODE5 fails.

I made some recent changes to not have to specify the list of keys to request but get it from all the registered sensors. Now, one only needs to change yaml to add new sensors.

I tried ALRMALRT and I also got 0. Based on the manual this is likely to be 0 or 1. Even if it's a count, I don't see much benefit for automations to know the exact count since we typically need to notify us if there is any alert and I assume we will immediately take action before new alerts appear. So I'd recommend going with a binary sensor. I think it's safe to add it now for all.

Reply:
Oh, awesome. I guess this change makes no sense at this point, then? I will close this PR.